failed to build kf5-konsole 14.11.97, require kinit, kdelibs4support

```
/private/tmp/kf5-konsole-0Mq2Fo/konsole-14.11.97$ cmake . -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/kf5-konsole/14.11.97 -DCMAKE_BUILD_TYPE=None -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DCMAKE_CXX_FLAGS='-D_DARWIN_C_SOURCE' -DCMAKE_INSTALL_BUNDLEDIR=/Applications/KDE
CMake Warning at /usr/local/opt/kf5-extra-cmake-modules/share/ECM/find-modules/FindKF5.cmake:77 (find_package):
  Could not find a package configuration file provided by "KF5Init" with any
  of the following names:

    KF5InitConfig.cmake
    kf5init-config.cmake

  Add the installation prefix of "KF5Init" to CMAKE_PREFIX_PATH or set
  "KF5Init_DIR" to a directory containing one of the above files.  If
  "KF5Init" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:31 (find_package)


-- Could NOT find KF5Init: found neither KF5InitConfig.cmake nor kf5init-config.cmake 
CMake Warning at /usr/local/opt/kf5-extra-cmake-modules/share/ECM/find-modules/FindKF5.cmake:77 (find_package):
  Could not find a package configuration file provided by
  "KF5KDELibs4Support" with any of the following names:

    KF5KDELibs4SupportConfig.cmake
    kf5kdelibs4support-config.cmake

  Add the installation prefix of "KF5KDELibs4Support" to CMAKE_PREFIX_PATH or
  set "KF5KDELibs4Support_DIR" to a directory containing one of the above
  files.  If "KF5KDELibs4Support" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:31 (find_package)


-- Could NOT find KF5KDELibs4Support: found neither KF5KDELibs4SupportConfig.cmake nor kf5kdelibs4support-config.cmake 
CMake Error at /usr/local/Cellar/cmake/3.2.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:138 (message):
  Could NOT find KF5 (missing: Init KDELibs4Support) (found version "5.8.0")
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.2.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:374 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/opt/kf5-extra-cmake-modules/share/ECM/find-modules/FindKF5.cmake:113 (find_package_handle_standard_args)
  CMakeLists.txt:31 (find_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/kf5-konsole-0Mq2Fo/konsole-14.11.97/CMakeFiles/CMakeOutput.log".
See also "/tmp/kf5-konsole-0Mq2Fo/konsole-14.11.97/CMakeFiles/CMakeError.log".
```

--- a/kf5-konsole.rb
+++ b/kf5-konsole.rb
@@ -44,6 +44,8 @@ class Kf5Konsole < Formula
   depends_on "haraldf/kf5/kf5-knotifyconfig"
   depends_on "haraldf/kf5/kf5-kparts"
   depends_on "haraldf/kf5/kf5-kross"
+  depends_on "haraldf/kf5/kf5-kinit"
+  depends_on "haraldf/kf5/kf5-kdelibs4support"
 
   def patches
     DATA

